### DISCUSSION TITLE
Suggestion

### DISCUSSION THREAD

**Discussion by Sasidharan1984** [m4mallu/clonebot#62] [Ideas]
Will you please add option to forward only files containing allowlist words. ie, the bot only clone the file, if the filename contains a word in the allowlist words.

**Comment by m4mallu** [comment]
There are no such words like **White Listed Words** in a <s>global perspective</s>, because  unparliamentary words are not globally defined. So, as a global repository like this cannot be configured with this concept.

**Reply by Sasidharan1984** [reply]
Apologies for my English. I have edited it.
